cosmos/noble.json

```diff
@@ -38,19 +38,21 @@
       "coinGeckoId": "usd-coin",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/noble/uusdc.png",
       "gasPriceStep": {
-        "low": 0,
-        "average": 0,
-        "high": 0.01
+        "low": 0.1,
+        "average": 0.1,
+        "high": 0.2
       }
     },
     {
       "coinDenom": "ATOM",
       "coinMinimalDenom": "ibc/EF48E6B1A1A19F47ECAEA62F5670C37C0580E86A9E88498B7E393EB6F49F33C0",
       "coinDecimals": 6,
+      "coinGeckoId": "cosmos",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cosmoshub/uatom.png",
```

Comment:
GitHub Actions is saying that this is an [invalid image URL](https://github.com/chainapsis/keplr-chain-registry/actions/runs/7100481549/job/19326658604#step:7:9). Does anyone know why?

Reply:
@HeesungB I'm also happy to move the `uatom.png` into `images/noble` if you want!

Reply:
@johnletey No.I think it would be nice to remove the 'coingecoid' and 'coinunknown' fields.

Reply:
I removed both fields in https://github.com/chainapsis/keplr-chain-registry/pull/314/commits/5d88708fa4a2b3c66ed611959fa4e714bc8f345c!